DataFormats/Portable/interface/PortableHostCollection.h

```diff
@@ -69,12 +70,11 @@ class PortableHostCollection {
   ConstBuffer const_buffer() const { return *buffer_; }
 
   // part of the ROOT read streamer
-  static void ROOTReadStreamer(PortableHostCollection* newObj, Layout& layout) {
+  static void ROOTReadStreamer(PortableHostCollection* newObj, Layout const& layout) {
     newObj->~PortableHostCollection();
     // use the global "host" object returned by cms::alpakatools::host()
     new (newObj) PortableHostCollection(layout.metadata().size(), cms::alpakatools::host());
     newObj->layout_.ROOTReadStreamer(layout);
-    layout.ROOTStreamerCleaner();
```

Comment:
This seems to be undoing the memory leak fix ?

Reply:
Yes. The fix needed to be ported to the multi collection too (done, to be pushed)